src/test/java/api/loans/AgedToLostScheduledNoticesProcessingTests.java

```diff
@@ -248,11 +247,11 @@ void shouldStopSendingAgedToLostNoticesOnceItemIsClaimedReturned() {
 
     claimItemReturnedFixture.claimItemReturned(new ClaimItemReturnedRequestBuilder()
       .forLoan(agedToLostLoan.getLoanId().toString())
-      .withItemClaimedReturnedDate(ClockUtil.getDateTime()));
-    final DateTime firstRunTime = getAgedToLostDate(agedToLostLoan).plus(
-      TIMING_PERIOD.timePeriod());
+      .withItemClaimedReturnedDate(ClockUtil.getZonedDateTime()));
+
+    final ZonedDateTime firstRunTime = TIMING_PERIOD.plusDate(getAgedToLostDate(agedToLostLoan));
     scheduledNoticeProcessingClient.runLoanNoticesProcessing(
-      firstRunTime.plus(RECURRENCE_PERIOD.timePeriod()).plusMinutes(1));
+      RECURRENCE_PERIOD.plusDate(firstRunTime).plusMinutes(1));
```

Comment:
This is an example of why I don't like the current approach I've taken with the `Period.plusDate()` (and `Period.minusDate()`).

This is awkward and I don't know if we want to take a different design within the scope of this PR or to go forward with this as-is until there is time to implement the better design.

Reply:
> This is awkward and I don't know if we want to take a different design within the scope of this PR or to go forward with this as-is until there is time to implement the better design.

I thought you said in another comment that due to the deadline and in order to keep this pull request as small as possible, there weren't any other practical approaches we could take.

Did I misunderstand that?

Reply:
You did not misunderstand, but I wanted to document what/where the problem I am concerned with, provided a specific example.

With the PR being so huge, I wanted to make it easier to find this for reviewers sake, for ducmentation sake, for future reference.

Reply:
> You did not misunderstand, but I wanted to document what/where the problem I am concerned with is, providing a specific example.
> 
> With the PR being so huge, I wanted to make it easier to find this for reviewers sake, for ducmentation sake, for future reference.

Gotcha, thank you. I now understand the purpose of this comment.

Maybe it's worthwhile adding a comment to the code as well, to explain how the code got this way and why that isn't desirable.